moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -32,7 +32,7 @@ export const ProjectsListViewQueryConf = {
   noResultsMessage: "No projects found.",
   showPagination: true,
   pagination: {
-    rowsPerPageOptions: [5, 10, 25, 50, 100],
+    rowsPerPageOptions: [25, 50, 100, 250, 1000],
```

Comment:
i think you can remove all but the options `250` and `1000`. who's sizing down?

Reply:
i dunno! maybe i just wanted pages? removed them.